src/app/Console/Commands/PublishHeaderMetas.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+namespace Backpack\CRUD\app\Console\Commands;
+
+use Illuminate\Console\Command;
+use Illuminate\Support\Facades\File;
+use Illuminate\Support\Str;
+
+class PublishHeaderMetas extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'backpack:publish-header-metas';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Publishes the header metas and assets.';
+
+    /**
+     * Execute the console command.
+     */
+    public function handle()
+    {
+        $appName = $this->ask('Whats the application name ?', 'Backpack');
+        $appUrl = $this->ask('Whats the application url ?', config('app.url'));
+        $backpackPrefix = config('backpack.base.route_prefix');
+        $appColor = $this->ask('Whats the application color ?', '#605ca8');
```

Comment:
I don't think making the app color purple is good for most projects. Personally I think that should be something muted, that stands out of the way. Here's what we use for Backpack, I think it's a better default:

```suggestion
        $appColor = $this->ask('What is the application color?', '#161c2d');
```

![image](https://github.com/Laravel-Backpack/CRUD/assets/1032474/0b6b24b2-dc5b-47c1-84b5-26e6b53d6158)
 versus the bright purple (Backpack's primary color) which looks like this
![image](https://github.com/Laravel-Backpack/CRUD/assets/1032474/7e0aeb07-dd91-43e0-b3e7-798ce45f25a2)
or the muted purple that was here before:
![image](https://github.com/Laravel-Backpack/CRUD/assets/1032474/7d227f97-ecc3-4c4b-9fcf-cf7033850f96)